File: PRAK605-2510817220023-NURFITRIA RAMADHANI.c
#include <stdio.h>

int main() {
    int a;
    scanf("%d", &a);

    int A[20][20], B[20][20], C[20][20];

    printf("Matriks A\n");
    for (int i = 0; i < a; i++) {
        for (int j = 0; j < a; j++) {
            scanf("%d", &A[i][j]);
        }
    }
    printf("Matriks B\n");
    for (int i = 0; i < a; i++) {
        for (int j = 0; j < a; j++) {
            scanf("%d", &B[i][j]);
        }
    }
    printf("Matriks AXB\n");
    for (int i = 0; i < a; i++) {
        for (int j = 0; j < a; j++) {
            C[i][j] = 0;
            for (int k = 0; k < a; k++) {
                C[i][j] += A[i][k] * B[k][j];
            }
        }
    }
    for (int i = 0; i < a; i++) {
        for (int j = 0; j < a; j++) {
            printf("%d ", C[i][j]);
        }
        printf("\n");
    }
    return 0;
}

File: PRAK605-2510817220023-NURFITRIA RAMADHANI.py
a = int(input())

print("Matriks A")
A = []
for i in range(a):
    baris = list(map(int, input().split()))
    A.append(baris)

print("Matriks B")
B = []
for i in range(a):
    baris = list(map(int, input().split()))
    B.append(baris)

print("Matriks AXB")
C = [[0] * a for _ in range(a)]

for i in range(a):
    for j in range(a):
        for k in range(a):
            C[i][j] += A[i][k] * B[k][j]

for i in range(a):
    for j in range(a):
        print(C[i][j], end=" ")
    print()